frontend/src/components/maps/leaflet/LayersControl/LayersTree.tsx

```diff
@@ -164,7 +212,7 @@ const LayersTree: React.FC<{ items: TreeMenuItem[] }> = ({ items }) => {
           );
         } else {
           return (
-            <LayerNode key={node.key} onClick={node.onClick}>
+            <LayerNode key={node.key} id={node.key}>
```

Comment:
getting rid of the onClick is desired?

Reply:
Yep, the nested layers do not need an onClick